### PR TITLE
Add boot-disk-size option on instance creation

### LINKED ISSUE #14: add boot-disk-size option on instance creation
The script creates instances with default boot disk size(10GB), but there is no env variable to modify this value.
It would be handy to add the option 
--boot-disk-size=${DISK_BOOT_SIZE_GB} 
in the function create_cluster

### PR DESCRIPTION
**Pull request by DanSedov**
To fix: https://github.com/GoogleCloudPlatform/bdutil/issues/14

### CODE REVIEW

File: bdutil [hunk 1]
```diff
@@ -591,6 +599,15 @@ function run_sanity_checks() {
         >> ${OVERRIDES_FILE}
   fi
 
+  if (( ${MASTER_BOOT_DISK_SIZE_GB} > 0 && MASTER_BOOT_DISK_SIZE_GB < 10 )); then
+    logerror 'MATER_BOOT_DISK_SIZE_GB can be a minimum of 10.'
```
Comment (dennishuo) [inline bdutil]: s/MATER/MASTER/

File: bdutil [hunk 2]
```diff
@@ -591,6 +599,15 @@ function run_sanity_checks() {
         >> ${OVERRIDES_FILE}
   fi
 
+  if (( ${MASTER_BOOT_DISK_SIZE_GB} > 0 && MASTER_BOOT_DISK_SIZE_GB < 10 )); then
```
Comment (dennishuo) [inline bdutil]: Might as well let this check also catch negative numbers

**Comment by DanSedov** [comment]
Addressed commends

**Comment by dennishuo** [comment]
LGTM

**Pull request merged**